app/assets/stylesheets/application.scss

```
@@ -123,3 +123,8 @@
   color:#27c0fb;
   font-size: 18px;
 }
+
+input[readonly] {
+  border-width: 1px;
+  border-style: solid;
```

```suggestion
  border-style: solid;
  background: #f4f4f4;
```

and get rid of the redundant `text-not-editable`